tigerpath/majors_and_certificates/majors/MUS_2018.json

```
@@ -0,0 +1,165 @@
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-music",
+    "https://music.princeton.edu/program-study"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Wendy Heller",
+      "email": "wbheller@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Juri Seo",
+      "email": "juri@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "MUS 105, MUS 106, and a 200-level Culture & Criticism course. Students who place out of MUS 105 by demonstrating equivalent knowledge in an exam given by the music faculty are required to take an additional elective to meet the minimum requirement of 11 courses in the major.",
+      "req_list": [
+        {
+          "name": "Required",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "course_list": [
+            "MUS 105",
+            "MUS 106"
+          ]
+        },
+        {
+          "name": "Culture & Criticism",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": null,
+          "course_list": [
+            "MUS 230",
+            "MUS 232",
+            "MUS 234",
+            "MUS 236",
+            "MUS 238",
+            "MUS 240",
+            "MUS 242",
+            "MUS 248",
+            "MUS 250",
+            "MUS 258",
+            "MUS 260",
+            "MUS 262",
+            "MUS 264",
+            "MUS 265",
+            "MUS 270"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Culture & Criticism",
+      "max_counted": 1,
+      "min_needed": 2,
+      "explanation": "Two Culture & Criticism courses.",
+      "course_list": [
+        "MUS 230",
+        "MUS 232",
+        "MUS 234",
+        "MUS 236",
+        "MUS 238",
+        "MUS 240",
+        "MUS 242",
+        "MUS 248",
+        "MUS 250",
+        "MUS 258",
+        "MUS 260",
+        "MUS 262",
+        "MUS 264",
+        "MUS 265",
+        "MUS 270",
+        "MUS 325",
+        "MUS 334",
+        "MUS 343",
+        "MUS 350",
+        "MUS 359",
+        "MUS 434",
+        "MUS 435"
+      ]
+    },
+    {
+      "name": "Materials & Making",
+      "max_counted": 1,
+      "min_needed": 3,
+      "explanation": "Three Materials & Making courses.",
+      "course_list": [
+        "MUS 209",

```

Also missing the first three courses here. What happened?
```suggestion
        "MUS 204",
        "MUS 205",
        "MUS 206",
        "MUS 209",
```
([reference](https://music.princeton.edu/materials-and-making))